Paccy10/ampersand_app_api
# Feature(batteries): create battery

**What does this PR do?**

- Create battery

**Description of Task to be completed?**

- Register a new battery

**How should this be manually tested?**

- Checkout to branch `ft-create-battery`
- Run the app with `flask run`
- Send a `POST` request to `/batteries` with the required payload
- A new battery record should be added to the database

**Any background context you want to provide?**

- N/A

**What are the relevant pivotal tracker stories?**

- N/A

**Screenshots (if appropriate)**

- N/A

**Questions:**

- N/A
